### PR TITLE
Updating data

### PR DESCRIPTION
**Pull request by EthanHogan**
updated data and schema to add the data that was missing for my component.

### CODE REVIEW

File: db/db.js [hunk 1]
```diff
@@ -21,13 +21,16 @@ const { Schema } = mongoose;
 const productSchema = new Schema({
   id: Number,
   product_name: String,
-  prime_pic: String,
```
Comment (EthanHogan) [inline db/db.js]: Realized I did not need any data about the images for my component

File: db/db.js [hunk 2]
```diff
@@ -21,13 +21,16 @@ const { Schema } = mongoose;
...
   price: String,
   seller: String,
   rating: Number,
   shipping: Boolean,
   condition: Boolean,
   availability: Number,
+  review_count: Number,
+  sale_count: Number,
+  sellerRating_count: Number,
+  seller_feedback: String,
```
Comment (EthanHogan) [inline db/db.js]: added this missing data to database

Comment (Xenolithes) [reply db/db.js]: Looks Good!
Are you planning on working with miles on getting the same seller rating and feedback?
Beyond that looks good to merge!

Comment (EthanHogan) [reply db/db.js]: I think the only data I will be getting from miles is the number of reviews, which is why they are all 0 for now. Everything else will be randomized!

**Pull request merged**